logs/app.log

```diff
@@ -0,0 +1,505 @@
+2022-10-03 10:36:05,889 INFO c.e.L.BookLibraryApplication [main] Starting BookLibraryApplication using Java 17.0.4.1 on DESKTOP-8GEMCRD with PID 7808 (C:\Users\stops\OneDrive\Робочий стіл\Library\target\classes started by stops in C:\Users\stops\OneDrive\Робочий стіл\Library)
```

Comment:
The log files are not considered a part of the source code, and thus could be removed from the PR